configuration.revanced.json

```diff
@@ -145,6 +145,46 @@
 					}
 				}
 			}
+		},
+		{
+			"includes": {
+				"channels": [952946952348270626, 953965039105232906],
+				"match": [
+					"(?i)(((tg|telegram|downloaded|download|revanced|channel).*){2,})"
```

Comment:
This would conflict with the other response regarding downloads.

Reply:
so can i just exclude some words or add additional one to the download matches here